recipe/meta.yaml

```diff
@@ -25,14 +25,8 @@ requirements:
     - {{ compiler('cxx') }}
     - cmake {{ cmake }}
     - make
-    # Use pins to control cos6/cos7 match
```

Comment:
These changes should probably be in another PR.
We are ready to merge the hw cap updates, but not these changes.

Reply:
Sure, reverted that commit.

Reply:
ok thanks!